Server/src/lobbycoordinator.erl

```diff
@@ -0,0 +1,100 @@
+-module(lobbycoordinator).
+-export([init/0]).
+%%Version 1.0
+%%Authors: Sebastian Fransson, Tim Jonasson
+
+%Initializes the lobby
+init() -> 
+  loop([], 0).
+  
+%the loop keeps track of the rooms and handles lobby requests.
+loop(Rooms, Lobby_increment) ->
+  receive 
+    %Creates a lobby and saves the monitor reference as well as lobby info to the list of rooms.
+    {Creator_Socket, create_lobby, Password} -> 
+	  % Spawn the process and save the Pid.
+	  Pid = spawn(fun () -> lobby:init(Creator_Socket, Password, Lobby_increment) end),
+	  % Save the reference so that we can use it later for demonitoring.
+      Ref = monitor(process, Pid),
```

Comment:
indentation is ripped

Reply:
That seems to be an issue with github, looks normal in the accual code

Reply:
^+1